### PR TITLE
local tools changes

### PR DESCRIPTION
**Pull request by wli3**
- Change manifest name to dotnet-tools.json

- Make isRoot required

- Remove redundant manifest file path in text

- Extra probing of .config folder

Since this text is indented under an error with manifest full path already

### CODE REVIEW

**Comment by wli3** [comment]
@dotnet-bot test https://github.com/dotnet/cli/pull/10188

**Comment by wli3** [comment]
@dotnet-bot test ubuntu.18.04 x64 Debug Build please

File: src/dotnet/ToolManifest/LocalizableStrings.resx [hunk 1]
```diff
@@ -129,7 +129,7 @@
 {1}</value>
   </data>
   <data name="ManifestVersion0" xml:space="preserve">
-    <value>Manifest version 0 is not supported. Manifest file path: {0}</value>
+    <value>Manifest version 0 is not supported.</value>
```
Comment (livarcocc) [inline src/dotnet/ToolManifest/LocalizableStrings.resx]: Why did we decide to remove the extra info?

Comment (wli3) [reply src/dotnet/ToolManifest/LocalizableStrings.resx]: it is redundant. It should be like the following 
```
Invalid manifest file. Path c:\dir\dotnet-tool.json:
  Manifest version 0 is not supported.  **removeing Path c:\dir\dotnet-tool.json**
```

**Pull request merged**